packages/components/lint.diff.js

```diff
@@ -0,0 +1,65 @@
+
+const execa = require('execa');
+
+const fileExtensions = ['.tsx', '.ts'];  // linted file types
```

Comment:
Yeah the gitignore will prevent from linting anything else.  This will just filter down the list of files to be passed to the lint command.  Maybe redundant, just doing an early filter to keep down the list size.